### ISSUE TITLE
Please add support for ios15 and Xcode13

### ISSUE THREAD

**Issue by sauravgarg65** [johnno1962/HotReloading#33]

**Comment by johnno1962** [comment]
Hi, what is the problem you're having? It should support ios15 and Xcode 13 already.

**Comment by sauravgarg65** [comment]
Command CompileSwiftSources failed with a nonzero exit code
This is an issue in Xprobe Swift

**Comment by johnno1962** [comment]
Very strange. Can you tell me the exact error message? What exact version of Xcode is this please?

**Comment by sauravgarg65** [comment]
Version 13.1 (13A1030d), it's not working for a particular project, but was working earlier.

**Comment by johnno1962** [comment]
You're not giving me enough information to work out what the problem is. I'm using Xcode 13.1 myself. Can you cut and paste the error message or a screenshot please? Have you tried a build clean?

**Comment by saim80** [comment]
@sauravgarg65 Is it build issue for one of the dependencies? In my case, my environment wasn't set up properly such that command line swift command used the macOS command line tools without any SDKs. All I had to do was running `xcode-select` command to make the swift command from Xcode.app as the default which will properly link SDKs while building dependencies. I'm not 100% sure but this is just one thing I noticed. I am using Xcode 13 without an issue.
